setup-files/Gerald-README.md

```diff
@@ -0,0 +1,63 @@
+# Gerald Usage Documentation
+
+Gerald is the project described under [ADR 356](https://docs.google.com/document/d/1TDE_nmrV3vuGi54HtC8X7irSMwTTcc9p83cuhH4kB6Y/edit#heading=h.zcx77itbdtis). The goal was to port over as much of the used Phabricator Herald functionality into Github Actions as possible.
+
+This is the [JIRA Epic](https://khanacademy.atlassian.net/browse/WEB-2448) to track the progress of the Gerald project.
```

Comment:
I think eventually we'll want to remove some of the Khan Academy-specific links, if folks are interested in using this outside of KA, but for now this is ok!